src/app/story/directives/player.component.ts

```diff
@@ -46,6 +50,7 @@ export class PlayerComponent implements OnDestroy, DoCheck {
         }
       });
     });
+    auth.token.subscribe(token => this.authToken = token);
```

Comment:
I think you'll want to look in app.component.spec.ts for an example of how to mock this token in a spec.

Reply:
thanks!